site/contents/write_ups/2022/16-06-22_writeup.md

```diff
@@ -0,0 +1,88 @@
+# Data Ethics Club: [Automating Inequality](https://www.youtube.com/watch?v=ADYxJU0WyJA)
+<!--Please don't edit the info panel below-->
+
+
+```{admonition} What's this? 
+This is summary of Thursday 16th June's Special Data Ethics Club discussion for the JGI's Data Week, where we discussed the video [Automating Inequality](https://www.youtube.com/watch?v=ADYxJU0WyJA) featuring Virginia Eubanks.
+
+The summary was written by Huw Day, who tried to synthesise everyone's contributions to this document and the discussion. "We" = "someone at Data Ethics Club". 
+Nina Di Cara and Natalie Thurlby helped with the final edit.
+
+```
+
+```{image} ../../../../images/AutomatingIneq.png
+:width: 200px
+:align: center
+:alt: Gru's Plan Meme, "Let the Algorithm decide exam grades" "People won't complain until it affects them" "Even Middle class people are affected". Meme made by Euan Bennet.
+```
+
+## Discussion
+
+As part of the Jean Goulding Institute's Data Week conference, we put on a special edition of Data Ethics Club, inviting our usual members as well as anyone attending Data Week to attend. Rather than give reading for the session beforehand, we watched a short video in the session. This video featured Virginia Eubanks talking about some of the ideas in her book (one our favourites here at DEC), Automating Inequality.
```

Comment:
```suggestion
As part of the [Jean Golding Institute's Data Week](http://www.bristol.ac.uk/golding/get-involved/data-week/) conference, we put on a special edition of Data Ethics Club, inviting our usual members as well as anyone attending Data Week to attend. Rather than give reading for the session beforehand, [we watched a short video in the session](https://www.youtube.com/watch?v=ADYxJU0WyJA). This video featured Virginia Eubanks talking about some of the ideas in her book (one our favourites here at DEC), Automating Inequality.
```